panel/src/components/Forms/Input/TagsInput.vue

```diff
@@ -247,6 +247,10 @@ export default {
           break;
       }
 
+      if (nextIndex < 0) {
+        nextIndex = 0;
+      }
```

Comment:
Make ensure not negative index

Reply:
Maybe we should add a similar check for not getting an index higher than the length.